app/controllers/mixins/actions/provider_actions/mass_transform.rb

```diff
@@ -1,56 +0,0 @@
-module Mixins
-  module Actions
-    module ProviderActions
-      module MassTransform
-        def vm_transform_mass
-          dialog = Dialog.find_by!(:label => 'Transform VM')
```

Comment:
Does this dialog still exist?

Reply:
No it was removed in https://github.com/ManageIQ/manageiq-providers-ovirt/pull/577